File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/allergy/d/AllergyDto.java
package iih.ci.ord.dto.allergy.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 实体 DTO数据 
 * 
 */
public class AllergyDto extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 过敏史
	 * @return String
	 */
	public String getId_patal() {
		return ((String) getAttrVal("Id_patal"));
	}
	/**
	 * 过敏史
	 * @param Id_patal
	 */
	public void setId_patal(String Id_patal) {
		setAttrVal("Id_patal", Id_patal);
	}
	/**
	 * 患者id
	 * @return String
	 */
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}
	/**
	 * 患者id
	 * @param Id_pat
	 */
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	/**
	 * 过敏史分类名称
	 * @return String
	 */
	public String getName_altp() {
		return ((String) getAttrVal("Name_altp"));
	}
	/**
	 * 过敏史分类名称
	 * @param Name_altp
	 */
	public void setName_altp(String Name_altp) {
		setAttrVal("Name_altp", Name_altp);
	}
	/**
	 * displaynam4
	 * @return String
	 */
	public String getName_alcla() {
		return ((String) getAttrVal("Name_alcla"));
	}
	/**
	 * displaynam4
	 * @param Name_alcla
	 */
	public void setName_alcla(String Name_alcla) {
		setAttrVal("Name_alcla", Name_alcla);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/allergy/d/AllergyDto.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.allergy.d
{
    /// <summary>
    /// AllergyDto 的摘要说明。
    /// </summary>
    public class AllergyDto : BaseDTO {

        public AllergyDto() {
 
        }

        /// <summary>
        /// 过敏史
        /// </summary>
		public string Id_patal {
            get { return getAttrVal<string>("Id_patal",null); }
            set { setAttrVal<string>("Id_patal", value); }
        }

        /// <summary>
        /// 患者id
        /// </summary>
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }

        /// <summary>
        /// 过敏史分类名称
        /// </summary>
		public string Name_altp {
            get { return getAttrVal<string>("Name_altp",null); }
            set { setAttrVal<string>("Name_altp", value); }
        }

        /// <summary>
        /// displaynam4
        /// </summary>
		public string Name_alcla {
            get { return getAttrVal<string>("Name_alcla",null); }
            set { setAttrVal<string>("Name_alcla", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_patal", "Id_pat", "Name_altp", "Name_alcla"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.allergy.d.AllergyDto";
        }
    }
}
